qrenderdoc/Windows/Dialogs/PerformanceCounterSelection.cpp

```diff
@@ -438,7 +438,7 @@ void PerformanceCounterSelection::Load()
         selectedCounters.insert(m_UuidToCounter[uuid]);
       }
 
-      SetSelectedCounters(selectedCounters.toList());
+      SetSelectedCounters(selectedCounters.values());
```

Comment:
What's the reason for this change? Checking the `QSet` docs, `toList()` is equivalent to `values()` and both have been around since our minspec (Qt 5.6).

Reply:
When building against more recent versions of QT, toList is marked as `deprecated` which will break the build due to `Werror`

Reply:
This breaks downstream packaged builds for linux distros like Arch for example (or any other system that has a more recent QT installed)

Reply:
`Werror` is not enabled in release builds for this exact reason, so that downstream packaged builds with different compiler setups don't get blocked when warnings appear. If any downstream package is having trouble, they should fix that instead by building a release build.

On top of that those Qt deprecation warnings should be disabled. In `qrenderdoc.pro` there's:

```
DEFINES += QT_NO_DEPRECATED_WARNINGS
```

Which I had to add because there was a ridiculous situation where something was deprecated when its replacement was only added in the immediate previous version, leaving it a mess to try and support a large old and new Qt versions.

Reply:
OK should I revert this then even though the minspec supports it?

Reply:
I'm fine with the change if it's needed for something, but I wanted to know why it was needed first. It seems like things should work with `toList`? Even on a dev build those warnings should be disabled and if they're not then other things would break. If so then all things being equal I'd prefer to revert it.

Reply:
Reverted. I just checked the release build and indeed, this change specifically isn't needed to fix the build

Reply:
Does it block a dev build? It shouldn't, because of the `QT_NO_DEPRECATED_WARNINGS`, so if that's a problem then it should still be investigated separately.